examples/byte-type/byte_type.bal

```diff
@@ -0,0 +1,11 @@
+import ballerina/io;
+
+public function main() {
+    // Byte values ranging from 0 to 255
+    byte b = 255;
+    io:println(b);
+
+    // Byte values can be assigned to int
```

Comment:
```suggestion
    // Since the set of possible `byte` values is a subset of `int` values,
    // the `byte` type is a subtype of the `int` type.
```